packages/pangea-sdk/pangea/services/redact.py

```diff
@@ -260,6 +262,8 @@ def redact(
             rulesets (list[str], optional): An array of redact rulesets short names
             return_result(bool, optional): Setting this value to false will omit the redacted result only returning count
             redaction_method_overrides: A set of redaction method overrides for any enabled rule. These methods override the config declared methods
+            llm_request: Boolean flag to enable FPE redaction for LLM requests
```

Comment:
```suggestion
            llm_request (bool, optional): Boolean flag to enable FPE redaction for LLM requests
```

Reply:
Docs generator can infer this from the types for a while now, so this sort of thing is mostly useless AFAIK.